test/property/arbitraries/canister_arb.ts

```diff
@@ -168,6 +170,7 @@ function generateSourceCode(
     `;
 
     return /*TS*/ `
+        ${inspectMessageImportHack === true ? `import { acceptMessage, methodName, msgCaller } from 'azle';` : ''}
```

Comment:
Oh I see it's all coming together. Yeah, I guess let's just talk about it. Is is just that all of other imports from from azle/experimental?